Sources/docc/DocCDocumentation.docc/other-formatting-options.md

```diff
@@ -0,0 +1,108 @@
+
+## Overview
+
+DocC offers support for various types of elements such as tables, notes, asides, and special characters.
+
+### Add Tables of Data
+
+To create a table, start a new paragraph and use dashes (`-`) to define 
+columns, and pipes (`|`) as column separators. Construction of a table requires 
+at least three rows: 
+
+* A row that contains the column names.
+* A row with cells that contain only dashes (`-`); three or more for each 
+column.
+* One or more rows of content.
+
+```markdown
+| Sloth speed  | Description                           |
+| ------------ | ------------------------------------- | 
+| `slow`       | Moves slightly faster than a snail.   |
+| `medium`     | Moves at an average speed.            |
+| `fast`       | Moves faster than a hare.             |
+| `supersonic` | Moves faster than the speed of sound. |
+```
+There's no need to impose a column width. A column determines its width 
+using the contents of its widest cell. You can also omit the leading 
+and trailing pipes.
+
+```markdown
+Sloth speed | Description
+--- | ---
+`slow` | Moves slightly faster than a snail.
+`medium` | Moves at an average speed.
+`fast` | Moves faster than a hare.
+`supersonic` | Moves faster than the speed of sound.
+```
+
+Both examples result in the same output:
+
+| Sloth speed | Description |
+| ---: | :--- |
+| `slow` | Moves slightly faster than a snail. |
+| `medium` | Moves at an average speed. |
+| `fast` | Moves faster than a hare. |
+| `supersonic` | Moves faster than the speed of sound. |
+
+> Note: Table cells don't support multiple paragraphs, lists, or code 
+listings.
+
+The text of a table cell can use the same style attributes as other text, and 
+include links to other content, including symbols. 
+
+### Add Notes and Other Asides
+
+There may be circumstances when you want to get the reader's attention to 
+provide additional advice, or to warn them about common errors or requisite 
+configuration. For those situations, use an aside. 
+
+DocC supports the following types of asides:
+
+| Type | Usage |
+| ----- | ------ |
+| Note | General information that applies to some users. |
+| Important | Important information, such as a requirement. |
+| Warning | Critical information, like potential data loss or an irrecoverable state. |
+| Tip | Helpful information, such as shortcuts, suggestions, or hints. |
+| Experiment | Instructional information to reinforce a learning objective, or to encourage developers to try out different parts of your framework. |
+
+To create an aside, begin a new line with a greater-than symbol (`>`), add a space, 
+the type of the aside, a colon (`:`), and the content of the aside.
+
+```markdown
+> Tip: Sloths require sustenance to perform activities.
+```
+
+For the example above, DocC renders the following aside:
+
+> Tip: Sloths require sustenance to perform activities.
+
+The text of an aside can use the same style attributes as other text, and 
```

Comment:
I think it's important to note that an aside can only contain a single line of text (meaning, a single paragraph). Asides don't provide support for multiple paragraphs, and you can't include lists, code blocks, or images within an aside.

Reply:
Added in https://github.com/apple/swift-docc/pull/742/commits/0d0651d7a625986ec00e5b29ecdeae51053d2109